database/migrations/20220610074055-associate-user-creator.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn(
+      "sites", // name of Source model
+      "creator_id", // name of column we're adding
+      {
+        type: Sequelize.BIGINT,
+        allowNull: false,
+        references: {
+          model: "users",
+          key: "id",
+        },
+        onUpdate: "CASCADE",
+        onDelete: "CASCADE",
+      }
+    )
```

Comment:
Actually, if a user is deleted, I think we might not want to cascade? We should still preserve the record of the site